peer.go

```diff
@@ -381,7 +383,22 @@ func (peer *Peer) OnCTRLMsg(msg webrtc.DataChannelMessage) {
 			Logger.Warn("Got an add_pane commenad with no rows or cols")
 		}
 
-		pane, err := NewPane(a.Command, peer, ws, a.Parent)
+		if a.Command[0] == "*" {
+			shell, err := loginshell.Shell()
+			if err != nil {
+				Logger.Warnf("Failed to determine user's shell: %v", err)
+				a.Command[0] = "bash"
+			} else {
+				Logger.Infof("Using %s for shell", shell)
+				a.Command[0] = shell
+			}
+		}
+		dirname, err := os.UserHomeDir()
+		if err != nil {
+			Logger.Warnf("Failed to determine user's home directory: %v", err)
+			dirname = "/"
+		}
+		pane, err := NewPane(append([]string{"env", fmt.Sprintf("HOME=%s", dirname)}, a.Command...), peer, ws, a.Parent)
```

Comment:
Nice composition of the first param to to NewPane but the line is too long. Please add

```
  cmd := append([]string{"env", fmt.Sprintf("HOME=%s", dirname)}, a.Command...)
```